crates/forge/tests/e2e/fuzzing.rs

```diff
@@ -6,7 +6,7 @@ use shared::test_utils::output_assert::{assert_stderr_contains, assert_stdout_co
 fn fuzzing() {
     let temp = setup_package("fuzzing");
 
-    let output = test_runner(&temp).arg("fuzzing").assert().code(1);
+    let output = test_runner(&temp).arg("fuzzing::").assert().code(1);
```

Comment:
Why was this changed? Do we really want to run `fuzzing` module only?

Reply:
Yep or else we run  `fuzzing_integrationtest`